client/my-sites/domains/domain-search/index.jsx

```diff
@@ -107,12 +110,20 @@ class DomainSearch extends Component {
 	}
 
 	componentDidUpdate( prevProps ) {
+		if ( this.props.domainSidebarExperimentUser ) {
+			document.body.classList.add( 'is-domain-sidebar-experiment-user' );
+		}
```

Comment:
Something else just came up as I saw this: we'll also need to cover the opposite scenario: on update, if `domainSidebarExperimentUser` is false, we should likely remove the class, just like we do on unmount.

Reply:
Nice, [fixed here](https://github.com/Automattic/wp-calypso/pull/73017/commits/504282123d8ac50cb8978af6f669603f23727aba).